tests/WWT.Tests.props

```diff
@@ -2,8 +2,8 @@
   xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows'">net5;net48</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows'">net5</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows'">net5.0;net48</TargetFrameworks>
```

Comment:
These are good - surprised I didn't see issues before. There has been back and forth on this TFM (target framework moniker), so net5 may actually work. But the official guidance does say net5.0